packages/experimental/Shimmer/src/Shimmer.types.ts

```diff
@@ -0,0 +1,40 @@
+import { SvgProps, RectProps, CircleProps } from 'react-native-svg';
+import { ImageURISource } from 'react-native';
+
+
+export const shimmerName = 'Shimmer';
+
+interface ShimmerPropTokens {
+  gradientTintColor?: string;
+  toValue?: number;
+  duration?: number;
+  delay?: number;
+  shimmerTintColor?: string;
+  containerWidth?: number | string;
+  containerHeight?: number | string;
+}
+
+export interface ShimmerTokens extends ShimmerPropTokens {
+  viewBox?: string;
+}
+
+export interface ShimmerSlotProps extends  ShimmerPropTokens {
+  root: SvgProps;
+  rect: RectProps;
+  circle: CircleProps;
+}
```

Comment:
Maybe a big ask, but could we document what each of these parameters / tokens are? Either with comments above the field in the type, or in a markdown file at the root of this repo?

Reply:
Not at all, added documentation.